tests/Doctrine/Tests/DBAL/Functional/ExceptionTest.php

```diff
@@ -246,9 +246,10 @@ public function testConnectionExceptionSqLite($mode, $exceptionClass)
             $this->markTestSkipped("Only fails this way on sqlite");
         }
 
-        $filename = sprintf('%s/%s', sys_get_temp_dir(), 'doctrine_failed_connection.db');
+        $filename = sprintf('%s/%s', sys_get_temp_dir(), 'doctrine_failed_connection_'.$mode.'.db');
```

Comment:
We need different files per mode because on Windows sqlite keeps the file reference open and thus the db file cannot be removed in the same process.